app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -277,6 +345,55 @@ export default {
       }
       return false;
     },
+    isContactInformation() {
+      if (this.currentChat.id) {
+        const { is_contact_info_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
+    },
+    isAgentDropdown() {
+      if (this.currentChat.id) {
+        const { is_agent_dropdown_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
+    },
+    isTeamsDropdown() {
+      if (this.currentChat.id) {
+        const { is_teams_dropdown_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
+    },
+    isConversationLabels() {
+      if (this.currentChat.id) {
+        const { is_conv_labels_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
+    },
+    isConversationDetails() {
+      if (this.currentChat.id) {
+        const { is_conv_details_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
+    },
+    isContactAttributes() {
+      if (this.currentChat.id) {
+        const { is_contact_attributes_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
+    },
+    isPreviousConversation() {
+      if (this.currentChat.id) {
+        const { is_previous_conv_open: isOpen } = this.uiSettings;
+        return isOpen;
+      }
+      return false;
```

Comment:
This entire thing is repetition. @muhsin-k / @nithindavid Can you please guide @iamsivin to create a common method to take a flag from uiSettings so the sections are extensible?

Reply:
I have already added a comment